Makefile

```diff
@@ -1,32 +1,48 @@
-.PHONY: install dev clean check test format check-format lint
+.PHONY: install dev dev-req clean check test format check-format lint release release-build release-validate
 
 VIRTUAL_ENV = .venv
 DEV_BUILD_FLAG = $(VIRTUAL_ENV)/DEV_BUILD_FLAG
 NOTEBOOKS = $(shell find docs -type f -name '*.ipynb' -not -path '*/.ipynb_checkpoints/*')
 
-install:
+install: $(DEV_BUILD_FLAG)
+	echo "pyvizzu" > PKG.cfg
 	$(VIRTUAL_ENV)/bin/python setup.py install
+	echo "ipyvizzu" > PKG.cfg
+	$(VIRTUAL_ENV)/bin/python setup.py install
+	echo "stpyvizzu" > PKG.cfg
+	$(VIRTUAL_ENV)/bin/python setup.py install
+
+dev-req: $(DEV_BUILD_FLAG)
+	$(VIRTUAL_ENV)/bin/pip-compile --upgrade dev-requirements.in
 
 dev: $(DEV_BUILD_FLAG)
 
 $(DEV_BUILD_FLAG):
 	python3 -m venv $(VIRTUAL_ENV)
-	$(VIRTUAL_ENV)/bin/python setup.py install
+	$(VIRTUAL_ENV)/bin/pip install pip==22.0.4
```

Comment:
@nyirog Thanks, I will add upgrade.
I had a problem with the newer pip version, but I will check that is still needed before finish this draft, because I am changing a lot of thing about packaging.